apm-application-toolkit/apm-toolkit-trace/src/main/java/org/apache/skywalking/apm/toolkit/trace/ActiveSpan.java

```diff
@@ -26,10 +26,21 @@
  */
 public class ActiveSpan {
```

Comment:
How about we support direct log in this pull request?

Reply:
i'll add